mod.ts

```diff
@@ -1,3 +1,25 @@
+// Inspired: https://github.com/graphext/worker-vm/blob/6bb5634dab905c9d041fb0d83cb174f8f9a9a380/mod.ts#L9
+
+// Based on: https://github.com/denoland/deno/blob/main/cli/tsc/dts/lib.deno.ns.d.ts#L549
+type PermissionOptionsObject = {
```

Comment:
Is it possible to use `Deno.PermissionOptions`?
https://docs.deno.com/api/web/~/WorkerOptions#property_deno